src/components/AlertForm/index.tsx

```diff
@@ -0,0 +1,423 @@
+          <Col span={24} md={12}>
+            <Input
+              label="Type:"
+              control={control}
+              name="type"
+              defaultValue={initialValue?.type || type}
+              disabled
+            />
+          </Col>
+        </Row>
+
+        <span className={styles.subtitle}>preview</span>
+        <Collapse
+          expandIcon={({ isActive }) => (
+            <Button className={styles.collapseBtn} size="small">
+              {isActive ? <MinusOutlined /> : <PlusOutlined />}
+            </Button>
+          )}
+          bordered={false}
+          className={styles.collapse}
+          activeKey={activePanel}
+          onChange={(keys) => setActivePanel(keys[0])}
+        >
+          <Panel
+            className={styles.panel}
+            header={
+              <div
+                className={cn(
+                  !activePanel && !isMobile && styles.headerWrapperDots
+                )}
+              >
+                <Space className={styles.header} size={10} align="center">
+                  {renderTags(COLORS.measure, measures)}
+                  {!activePanel && (
+                    <>
+                      <span className={styles.tagLabel}>BY</span>
+                      {renderTags(COLORS.dimension, dimensions)}
+                      {renderTags(COLORS.timeDimension, timeDimensions)}
+                      <span className={styles.tagLabel}>IN</span>
+                      {renderTags(COLORS.segment, segments)}
+                      <span className={styles.tagLabel}>ORDERED BY</span>
+                      {renderTags(
+                        COLORS.order,
+                        order?.map((o) => `${o.name}.${o.order}`)
+                      )}
+                    </>
+                  )}
+                </Space>
+              </div>
+            }
+            key={"1"}
+          >
+            <Space
+              className={styles.collapseInner}
+              direction="vertical"
+              size={10}
+            >
+              {(dimensions || timeDimensions) && (
+                <Space size={9}>
+                  <span className={styles.tagLabel}>BY</span>
+                  {renderTags(COLORS.dimension, dimensions)}
+                  {renderTags(COLORS.timeDimension, timeDimensions)}
+                </Space>
+              )}
+              {segments && (
+                <Space size={9}>
+                  <span className={styles.tagLabel}>IN</span>
+                  {renderTags(COLORS.segment, segments)}
+                </Space>
+              )}
+
+              {order && (
+                <Space size={9}>
+                  <span className={styles.tagLabel}>ORDERED BY</span>
+                  {renderTags(
+                    COLORS.order,
+                    order?.map((o) => `${o.name}.${o.order}`)
+                  )}
+                </Space>
+              )}
+            </Space>
+          </Panel>
+        </Collapse>
```

Comment:
should be a component here.
Also please create several variants of data in the component to represent all states